tests/migration/online_migration/pre_migration.pm

```diff
@@ -24,6 +24,12 @@ sub check_or_install_packages {
             record_soft_failure('bsc#1197268', 'suseconnect-ng obsoletes zypper-migration-plugin in leap to sle migration');
             zypper_call('rm zypper-migration-plugin');
             zypper_call "in yast2-registration rollback-helper";
+            if (get_var('LEAP_TECH_PREVIEW')) {
```

Comment:
Maybe you can user one setting, only the LEAP_TECH_PREVIEW_REPO. WDYT?

Reply:
yes